Shoko.Server/Providers/TraktTV/TraktTVHelper.cs

```diff
@@ -286,11 +286,13 @@ public void RefreshAuthToken()
             settings.TraktTv.TokenExpirationDate = string.Empty;
 
             _logger.LogError(ex, "Error in TraktTVHelper.RefreshAuthToken");
+            return false;
```

Comment:
this one is unnecessary since both the last path in the above try block and this block both lead out of the try…catch statement, and would hit the return statement below.

Reply:
I had to add it because the IDE would give me an error that the return state is missing :)